test/event_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event'
+
+class EventTest < Minitest::Test
+  def test_its_max
+    curling = Event.new("Curling", [24, 30, 18, 20, 41])
+    assert_equal 41, curling.max_age
+  end
+
+  def test_its_min
+    curling = Event.new("Curling", [24, 30, 18, 20, 41])
+    assert_equal 18, curling.min_age
+  end
+
+  def test_its_avg
+    curling = Event.new("Curling", [24, 30, 18, 20, 41])
+    assert_equal 26.6, curling.average_age
+  end
+
+  def test_standard_dev
```

Comment:
Overall, these tests look pretty good.  Keep an eye on the names of the tests; they should read more like 'test_it_can_calculate_standard_deviation'